example/image-classification/train_imagenet.py

```diff
@@ -52,7 +52,11 @@
 
 # network
 import importlib
-net = importlib.import_module('symbol_' + args.network).get_symbol(args.num_classes)
+if args.network == 'inception-resnet-v2':
```

Comment:
@piiswrong   As I know,  I can't get the input_data_shape in the `get_symbol()`. So i pass the input_data_shape which generated by the `args.batch_size and args.data_shape`. If there is a way to get the input_data_shape in the `get_symbol()`, Pls let me know and i will change, Thanks